invenio_records_marc21/templates/invenio_records_marc21/records/helpers/details.html

```diff
@@ -6,14 +6,17 @@
   under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-{%- from "invenio_records_marc21/records/macros/detail.html" import show_subtitles, show_detail, show_publisher %}
+{%- from "invenio_records_marc21/records/macros/detail.html" import show_subtitles, show_detail, show_publisher, show_dissertation, show_location_and_access, show_index_term_genre %}
 
 {% set details = marcrecord %}
 <h4 class="ui header">{{ _('Details')}}</h4>
 <hr class="thin-line">
-<div class="top-bottom-padded">  
+<dd class="top-bottom-padded" id="marc21-landing-page-details">  
```

Comment:
why do you change that to <dd></dd>?

Reply:
I changed the tag name according to the RDM records landing page.

Reply:
don't do that in that particularly thing. please leave it as it was.